pkg/pipelinerun/description/description.go

```diff
@@ -97,6 +97,21 @@ STARTED	DURATION	STATUS
 {{- end }}
 {{- end }}
 
+{{decorate "workspaces" ""}}{{decorate "underline bold" "Workspaces\n"}}
+
+{{- if eq (len .PipelineRun.Spec.Workspaces) 0 }}
+ No workspaces
+{{- else }}
+ NAME	SUB PATH		WORKSPACE BINDING
+{{- range $workspace := .PipelineRun.Spec.Workspaces }}
+{{- if not $workspace.SubPath }}
+ {{ decorate "bullet" $workspace.Name }}	{{ "---" }}		{{ formatWorkspace $workspace }}
```

Comment:
I guess there are two tabs in between 2nd and 3rd column

Reply:
thanks for pointing out :+1: . Fixed